### PR TITLE
fix: missing client markdown capabilities

### PR DESCRIPTION
**Pull request by sumimakito**
**tl;dr** This pull request adds the missing `clientCapabilities` in the YAML worker to fix the broken support for `markdownDescription`.

It seems monaco-yaml does not set `clientCapabilities` upon calling `getLanguageService`, which is exported by yaml-language-server. This causes the [`doesSupportMarkdown`](https://github.com/redhat-developer/yaml-language-server/blob/f039273ee5b6974db5ad34b03bb24cfe09b64447/src/languageservice/services/yamlCompletion.ts#L1642-L1652) in yaml-language-server always return undefined and makes the `markdownDescription` not being correctly rendered in the completion item.

I can see `clientCapabilities` [is being set](https://github.com/redhat-developer/yaml-language-server/blob/f039273ee5b6974db5ad34b03bb24cfe09b64447/test/utils/testHelper.ts#L87) by the test helper in yaml-language-server with a constant imported from vscode-json-languageservice.

Similiar usage in jsonWorker in monaco-editor can be found here: https://github.com/microsoft/monaco-editor/blob/f6dc0eb8fce67e57f6036f4769d92c1666cdf546/src/language/json/jsonWorker.ts#L38

_This pull request replaces #248_

### CODE REVIEW

**Comment by remcohaszing** [comment]
Thanks! But why did you go with this approach instead of the one in #248? I actually like that one better at first glance.

**Comment by remcohaszing** [comment]
I merged #248 instead.
